Makefile

```diff
@@ -24,8 +26,67 @@ release:
 mrconfig:
 	./scripts/update_mrconfig.py
 
+muconfig:
+	mu group add salt-formulas --empty
+	mu register $(FORKED_FORMULAS_DIR)/*
+
 html:
 	make -C doc html
 
 pdf:
 	make -C doc latexpdf
+
+FORKED_FORMULAS_DIR=fformulas
+FORMULAS=`python3 -c 'import sys; sys.path.append("scripts");from update_mrconfig import *; print(*get_org_repos(make_github_agent(), "salt-formulas"), sep="\n")'| egrep 'salt-formula-' | sed 's/salt-formula-//'`
+
+scripts_prerequisites:
+	pip3 install -r scripts/requirements.txt
```

Comment:
No way, you don't want to let makefile silently mess your system when running eg. `make list`